packages/near-fast-auth-signer/webpack.common.config.js

```diff
@@ -1,12 +1,23 @@
 const childProcess = require('child_process');
+const fs = require('fs');
 const path = require('path');
 
 const { sentryWebpackPlugin } = require('@sentry/webpack-plugin');
+const dotenv = require('dotenv');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const webpack = require('webpack');
 
 // Execute Git command to get the current commit ID
 const commitHash = childProcess.execSync('git rev-parse --short HEAD').toString().trim();
+// Load environment variables from .env file or process.env
+const envFilePath = path.resolve(__dirname, '.env');
+const envConfig = fs.existsSync(envFilePath) ? dotenv.parse(fs.readFileSync(envFilePath)) : process.env;
```

Comment:
With this approach, if there is .env and also env variables being pass through commend line, would env from commend line gets ignored?

Reply:
Hey @hcho112 nope it will not get ignored, it will be included.
And it will also take precedence.